.github/workflows/CI.yml

```diff
@@ -682,6 +697,8 @@ jobs:
       - name: Set up terraform
         if: steps.e2etest-ec2-2.outputs.cache-hit != 'true'
         uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_wrapper: false
```

Comment:
Why is this being disabled?

Reply:
It would be failed when assigning  ```terraform output {{output variable}}``` to a variable since by using ```terraform_wrapper=true```, it will assign the variable the whole raw command instead of getting the output from it. More in depth read is [in this article ](https://blog.nillsf.com/index.php/2020/08/25/how-to-use-terraform-output-in-a-github-action/). Moreover, I tested this out with and without using this and seems the former works for me.